benchcab/internal.py

```diff
@@ -82,6 +82,88 @@
 # CABLE fixed C02 concentration
 CABLE_FIXED_CO2_CONC = 400.0
 
+# Contains the site ids for each met forcing file associated with an experiment
+# on modelevaluation.org
+MEORG_EXPERIMENTS = {
+    # List of site ids associated with the 'Five site test'
+    # experiment (workspace: NRI Land testing), see:
+    # https://modelevaluation.org/experiment/display/xNZx2hSvn4PMKAa9R
+    "five-site-test": [
+        "AU-Tum",
+        "AU-How",
+        "FI-Hyy",
+        "US-Var",
+        "US-Whs",
+    ],
+    # List of site ids associated with the 'Forty two site test'
+    # experiment (workspace: NRI Land testing), see:
+    # https://modelevaluation.org/experiment/display/urTKSXEsojdvEPwdR
+    "forty-two-site-test": [
+        "AU-Tum",
+        "AU-How",
+        "AU-Cum",
+        "AU-ASM",
+        "AU-GWW",
+        "AU-Ctr",
+        "AU-Stp",
+        "BR-Sa3",
+        "CA-Qfo",
+        "CH-Dav",
+        "CN-Cha",
+        "CN-Din",
+        "DE-Geb",
+        "DE-Gri",
+        "DE-Hai",
+        "DE-Tha",
+        "DK-Sor",
+        "FI-Hyy",
+        "FR-Gri",
+        "FR-Pue",
+        "GF-Guy",
+        "IT-Lav",
+        "IT-MBo",
+        "IT-Noe",
+        "NL-Loo",
+        "RU-Fyo",
+        "US-Blo",
+        "US-GLE",
+        "US-Ha1",
+        "US-Me2",
+        "US-MMS",
+        "US-Myb",
+        "US-NR1",
+        "US-PFa",
+        "US-FPe",
+        "US-SRM",
+        "US-SRG",
+        "US-Ton",
+        "US-UMB",
+        "US-Var",
+        "US-Whs",
+        "US-Wkg",
+    ]
+}
+
+
+def get_met_sites(experiment: str) -> list[str]:
+    '''Get a list of met forcing file basenames specified by an experiment
+
+    The `experiment` argument either specifies a key in `MEORG_EXPERIMENTS` or a site id
+    within the five-site-test experiment.
+
+    Assume all site ids map uniquely to a met file in MET_DIR.
+    '''
+
+    if experiment in MEORG_EXPERIMENTS["five-site-test"]:
+        # the user is specifying a single met site
+        return list(map(lambda path: path.name, MET_DIR.glob(f"{experiment}*")))
```

Comment:
```suggestion
        # Use next() to return the value from the `glob()` generator
        return [next(MET_DIR.glob(f"{experiment}*")).name]
```
I wonder if this syntax would be simpler. Not sure the syntax is 100% correct with the `)` so be careful if accepting on GitHub.

Reply:
Here are the changes: [ce4f1b9](https://github.com/CABLE-LSM/benchcab/pull/27/commits/ce4f1b9746af00fd361680afdb75a05a9fdc1808)